File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/system/DetailPaInfoFormGridVisitListInfo.java
package nta.med.data.model.ihis.system;

import java.sql.Timestamp;

public class DetailPaInfoFormGridVisitListInfo {
	private Timestamp comingDate;
    private String departmentName;
    private String typeName;
    private String doctorName;
    private Timestamp outDate;
	public DetailPaInfoFormGridVisitListInfo(Timestamp comingDate,
			String departmentName, String typeName, String doctorName,
			Timestamp outDate) {
		super();
		this.comingDate = comingDate;
		this.departmentName = departmentName;
		this.typeName = typeName;
		this.doctorName = doctorName;
		this.outDate = outDate;
	}
	public Timestamp getComingDate() {
		return comingDate;
	}
	public void setComingDate(Timestamp comingDate) {
		this.comingDate = comingDate;
	}
	public String getDepartmentName() {
		return departmentName;
	}
	public void setDepartmentName(String departmentName) {
		this.departmentName = departmentName;
	}
	public String getTypeName() {
		return typeName;
	}
	public void setTypeName(String typeName) {
		this.typeName = typeName;
	}
	public String getDoctorName() {
		return doctorName;
	}
	public void setDoctorName(String doctorName) {
		this.doctorName = doctorName;
	}
	public Timestamp getOutDate() {
		return outDate;
	}
	public void setOutDate(Timestamp outDate) {
		this.outDate = outDate;
	}
    
}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/System/DetailPaInfoFormGridVisitListInfo.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace IHIS.CloudConnector.Contracts.Models.System
{
    [Serializable]
    public class DetailPaInfoFormGridVisitListInfo
    {
        private String _comingDate;
        private String _departmentName;
        private String _typeName;
        private String _doctorName;
        private String _outDate;

        public String ComingDate
        {
            get { return this._comingDate; }
            set { this._comingDate = value; }
        }

        public String DepartmentName
        {
            get { return this._departmentName; }
            set { this._departmentName = value; }
        }

        public String TypeName
        {
            get { return this._typeName; }
            set { this._typeName = value; }
        }

        public String DoctorName
        {
            get { return this._doctorName; }
            set { this._doctorName = value; }
        }

        public String OutDate
        {
            get { return this._outDate; }
            set { this._outDate = value; }
        }

        public DetailPaInfoFormGridVisitListInfo() { }

        public DetailPaInfoFormGridVisitListInfo(String comingDate, String departmentName, String typeName, String doctorName, String outDate)
        {
            this._comingDate = comingDate;
            this._departmentName = departmentName;
            this._typeName = typeName;
            this._doctorName = doctorName;
            this._outDate = outDate;
        }

    }

}
